spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,98 @@
+<html>
+
+<style>
+  html * {
+    font-family: Roboto, Verdana, sans-serif;
+  }
+
+  body {
+    max-width: 50em;
+  }
+
+  li {
+    padding: 0.25em;
+  }
+
+  .panel {
+    margin: 1em;
+    padding: 1em;
+    border: 1px solid black;
+    border-radius: 5px;
+  }
+
+  .highlight {
+    background-color: #d6f5d6;
+  }
+
+
+</style>
+
+<head>
+  <title>Google Cloud Secret Manager Demo</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <link type="text/css" href="css/style.css" rel="stylesheet"/>
+</head>
+
+<body>
+<h1>Secret Manager Demo with Spring Cloud GCP</h1>
+
+<div class="panel">
+  <h3>Secret Manager Property Source</h3>
+  At the bootstrap phase, we loaded the following secret into the application context:
+  <br/>application-secret: [[${applicationSecret}]]
```

Comment:
The secret blends into the sentence. Maybe emphasize it a bit more?
```suggestion
  <br/><b>application-secret:</b> <i>[[${applicationSecret}]]</i>
```

Reply:
Done.